### DISCUSSION TITLE
Lab 4-1 LA/IO/IRQ問題

### DISCUSSION THREAD

**Discussion by v00000v** [bol-edu/HLS-SOC-Discussions#124] [Q&A] [answered]
請問user_proj_example.counter.v中的LA/IO/IRQ的相關訊號是要以caravel-soc_fpga/rtl/user/user_proj_example.counter.v中的方式宣告嗎?
如下圖
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/123727462/f920d6f8-9c0e-42d7-9752-117ec6cd351d)

**Comment by LingChiYang** [comment, marked as answer]
同學好

這些不會在lab4用到，之後的lab才會用
